gandi/cli/commands/domain.py

```diff
@@ -58,8 +57,31 @@ def info(gandi, resource):
 @pass_gandi
 def create(gandi, domain, duration, owner, admin, tech, bill, background):
     """Buy a domain."""
-    result = gandi.domain.create(domain, duration, owner, admin, tech, bill,
-                                 background)
+
+    # If domain is not available, no need to continue
+    if not domain:
+            return
+
+    user_handle = gandi.call('contact.info')['handle']
+    owner_ = owner or user_handle
+    admin_ = admin or user_handle
+    tech_ = tech or user_handle
+    bill_ = bill or user_handle
+
+    msg = """
+The domain {domain} is available. If you continue, registration will be attempted with the following parameters:
```

Comment:
Nice addition to display a summary here as we do when we created a vm. Maybe you can reuse the same output syntax, like * for each line, of information so we can separate important stuff and a /!\ for saying that it will be charged ?
Also check PEP8 for this string as it's very long and it will break.
A rewording may be necessary to reduce it, like saying that domain is available on one line, and the next a little bit shorter ?